lib/routes/content.js

```diff
@@ -215,7 +215,7 @@ const routes = application => {
             return types.form(merged, errors, config).then(form => {
               res.render('content/add', {
                 form,
-                action: `/${config.content.base}/${req.params.type}/${config.content.actions.save}`,
+                action: `/${config.content.base}/${req.params.type.toLowerCase()}/${config.content.actions.save}`,
```

Comment:
This all looks good. Just requesting that a few tests are written so we can catch this before it crops up again?